src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const row = document.querySelectorAll('tr');
+
+const firstCell = [...row][0].querySelectorAll('th');
+const lastCell = [...row][row.length - 1].querySelectorAll('th');
+const positionName = [...firstCell][1].textContent;
+
+const position = document.createElement('th');
+
+position.textContent = positionName;
+
+const lastUpperPosition = [...firstCell][4];
+const lastLowerPosition = [...lastCell][4];
+
+row[0].insertBefore(position, lastUpperPosition);
+
+row[row.length - 1]
+  .insertBefore(position.cloneNode(true), lastLowerPosition);
+
+for (let i = 1; i < [...row].length - 1; i++) {
```

Comment:
Why are you using a copy everywhere throughout your solution? You just need to find the column and copy it once. It is in this case that you operate only with the number in the cycle, so why is there a copy? 
And your solution can be simplify. Let's try to improve it.

take your `row` from 3 line

than use `forEach`

```
row.forEach((element) => {
  const copy = element.children[1];
  const grandpa = copy.parentElement.parentElement.tagName;
```

then you check then `grandpa` to know how to insert `th` or `td` and then `element.children[3].insertAdjacentHTML........`

easy-peasy)
fill free to ask in `fe_chat` if something is not clear